sinch/domains/verification/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+from typing import TypedDict
+
+
+class VerificationIdentity(TypedDict):
+    type: str
```

Comment:
The spec mentions that only `number` is accepted as a value for this property. Maybe you can give a hint to the user about valid values

Reply:
Why not. Literal type hint added.